Unittests fail in node 6.5.0

With Node 4 unittests fail:

 1) Suite .beforeAll() wraps the passed in function in a Hook adds it to _beforeAll:
 2) Suite .afterAll() wraps the passed in function in a Hook adds it to _afterAll:
 3) Suite .beforeEach() wraps the passed in function in a Hook adds it to _beforeEach:
 4) Suite .afterEach() wraps the passed in function in a Hook adds it to _afterEach:

All have a similar Error:
`AssertionError: expected '"before all" hook: fn' to be '"before all" hook'`

Reason:
In node 6.5.0 anomynous functions have a name when assigned to a variable:

```
var myFunc = function(){};
console.log(myFunc.name); // ->  'myFunc'
```

Prior to node 6.5.0 the function name would be empty

--- a/test/suite.js
+++ b/test/suite.js
@@ -136,7 +136,7 @@ describe('Suite', function(){
 
         this.suite._beforeAll.should.have.length(1);
         var beforeAllItem = this.suite._beforeAll[0];
-        beforeAllItem.title.should.equal('"before all" hook');
+        beforeAllItem.title.should.match(/^"before all" hook/);
         beforeAllItem.fn.should.equal(fn);
       });
 
@@ -171,7 +171,7 @@ describe('Suite', function(){
 
         this.suite._afterAll.should.have.length(1);
         var afterAllItem = this.suite._afterAll[0];
-        afterAllItem.title.should.equal('"after all" hook');
+        afterAllItem.title.should.match(/^"after all" hook/);
         afterAllItem.fn.should.equal(fn);
       });
       it('appends title to hook', function(){
@@ -205,7 +205,7 @@ describe('Suite', function(){
 
         this.suite._beforeEach.should.have.length(1);
         var beforeEachItem = this.suite._beforeEach[0];
-        beforeEachItem.title.should.equal('"before each" hook');
+        beforeEachItem.title.should.match(/^"before each" hook/);
         beforeEachItem.fn.should.equal(fn);
       });
 
@@ -240,7 +240,7 @@ describe('Suite', function(){
 
         this.suite._afterEach.should.have.length(1);
         var afterEachItem = this.suite._afterEach[0];
-        afterEachItem.title.should.equal('"after each" hook');
+        afterEachItem.title.should.match(/^"after each" hook/);
         afterEachItem.fn.should.equal(fn);
       });
 

